.github/workflows/build-ee.yml

```diff
@@ -32,16 +32,36 @@ jobs:
 
       - uses: actions/setup-python@v4
 
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y qemu binfmt-support
+
+      - name: Setup QEMU for multi-platform builds
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: all
+
+      - name: Cache Buildah layers
```

Comment:
@fridim I added cache layer here, it will improve the performance to rebuild the images